elyra/cli/pipeline_app.py

```diff
@@ -29,6 +29,9 @@
 
 
 def _get_runtime_type(runtime_config: str) -> str:
+    if not runtime_config:
```

Comment:
If `runtime_config` can now be `None`, we should technically change the type hint to: `runtime_config: Optional[str]`.  Same goes for the return type.